src/BenchmarkDotNet/Environments/Runtimes/CoreRuntime.cs

```diff
@@ -19,6 +19,7 @@ public class CoreRuntime : Runtime
         public static readonly CoreRuntime Core50 = new CoreRuntime(RuntimeMoniker.Net50, "net5.0", ".NET 5.0");
         public static readonly CoreRuntime Core60 = new CoreRuntime(RuntimeMoniker.Net60, "net6.0", ".NET 6.0");
         public static readonly CoreRuntime Core70 = new CoreRuntime(RuntimeMoniker.Net70, "net7.0", ".NET 7.0");
+        public static readonly CoreRuntime Mono70 = new CoreRuntime(RuntimeMoniker.Mono70, "mono7.0", "Mono with .NET 7.0");
```

Comment:
Not sure if this should be here or in `MonoRuntime`

Reply:
I agree, `MonoRuntime` would most likely be a better place.